aws/aws-util/src/main/java/io/atleon/aws/util/AwsConfig.java

```diff
@@ -22,15 +23,18 @@ public final class AwsConfig {
     /**
      * The type of {@link AwsCredentialsProvider} to use. The "default" Provider uses a "chain" of
      * Providers which are capable of sourcing {@link AwsCredentials} from various environmental
-     * locations. When set to "static", the Credentials must be explicitly provided by setting the
-     * Access Key ID and Secret Access Key.
+     * locations. The "web.identity.token.file" Provider loads temporary credentials from
+     * Java system properties or environment variables. When set to "static", the Credentials must be
+     * explicitly provided by setting the Access Key ID and Secret Access Key.
      */
     public static final String CREDENTIALS_PROVIDER_TYPE_CONFIG = CONFIG_PREFIX + "credentials.provider.type";
 
     public static final String CREDENTIALS_PROVIDER_TYPE_DEFAULT = "default";
 
     public static final String CREDENTIALS_PROVIDER_TYPE_STATIC = "static";
 
+    public static final String CREDENTIALS_PROVIDER_WEB_IDENTITY_TOKEN_FILE = "web.identity.token.file";
```

Comment:
Let's make this kebab-cased (i.e. `web-identity-token-file`) in order to be consistent with other possible config values (like `region`, where example values are `us-east-1`, `eu-west-2`, etc.)

Reply:
And let's make the variable name consistent with the others with `TYPE` in it, i.e. `CREDENTIALS_PROVIDER_TYPE_WEB_IDENTITY_TOKEN_FILE`